pwa/src/components/logs/logTable/logTable.tsx

```diff
@@ -1,60 +1,35 @@
 import * as React from "react";
```

Comment:
General feedback for this document: I get a lot of Typescript errors.

E.g. on line `212` I get the following error: `Property 'session' does not exist on type`.

Reply:
Session does not exist because there is none set on the dummy data. If there is a session value de ide should recognize it and remove the error.